src/components/widgets/mmu/MmuUnit.vue

```diff
@@ -287,6 +296,33 @@ export default class MmuUnit extends Mixins(BrowserMixin, StateMixin, MmuMixin)
     return `#${this.unitIndex + 1} ${name}`
   }
 
+  get printerSensors (): Sensor[] {
+    return this.$typedGetters['printer/getSensors']
+  }
+
+  get unitClimateInfo (): string {
+    const unit = this.unitDetails(this.unitIndex)
+
+    // Handle missing or quoted sensor name
+    const sensorName = unit.environmentSensor?.replace(/^"(.*)"$/, '$1')
+    if (!sensorName) return ''
+
+    const sensor = this.printerSensors.find(s => s.key === sensorName)
+    if (!sensor) return ''
+
+    const parts: string[] = []
+
+    if (sensor.temperature) {
```

Comment:
I assume you only want to exclude `null` and `undefined` and not `0`, so:

```suggestion
    if (sensor.temperature != null) {
```